src/components/Chip/Chip.js

```diff
@@ -1,15 +1,48 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { hightlightSubString } from "../../utils";
 
 import "./Chip.scss";
 
-const Chip = ({ value, lead = "", onDismiss, selected }) => {
+const Chip = ({
+  value,
+  lead = "",
+  onClick,
+  onDismiss,
+  selected,
+  subString = "",
+}) => {
+  // When user tabs over chip and presses Enter or Space key, chip will trigger
+  // click functionality
+  const onKeyPress = (e) => {
+    // charCode 32 = Space key
+    // charCode 13 = Enter key
+    if (e.charCode === 32 || e.charCode === 13) {
```

Comment:
Might be nice to defined these as consts, then you don't need the comments:

`if (e.charCode === SPACE_KEY || e.charCode === ENTER_KEY)`

is more readable.

Reply:
Fixed this by using `onKeyDown` event.. thanks.